### PR TITLE
more style fixes

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

- show red borders on invalid input fields (previously overridden by mistake)
- remove red alert icon on invalid fields

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/frontend/src/assets/scss/style.scss [hunk 1]
```diff
@@ -264,15 +264,23 @@ a,
...
       color: $bcgov-input-placeholder;
       opacity: inherit;
     }
   }
 
+  .form-control.is-invalid,
+  .was-validated .form-control:invalid {
+    background-image: none;
+  }
+
```
Comment (loneil) [inline app/frontend/src/assets/scss/style.scss:283]: I think we need to be really careful with this. Are there other background images used for other formio functionality that having a `background-image: none` could override? Like some other control using an icon that's neccessary and having it disappear on validation could be unintended behaviour.

What's the intent of removing this as well? It's a FormIO default isn't it. We've decided not to mess around with basic FormIO behaviour of the rendered controls.

Personally I'd rather stay away from removing things like this unless it's an absolute requirement.
@parc-jason you've played more with the actual formIO controls than I have.

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-76

File: app/frontend/src/assets/scss/style.scss (hunk 1 above)
Comment (TimCsaky) [reply app/frontend/src/assets/scss/style.scss:283]: yes. good point. maybe we can just apply this to select fields.
this was done to address the issue with the red circle icon appearing over the dropdown arrow. 
image

Comment (parc-jason) [reply app/frontend/src/assets/scss/style.scss:283]: @loneil  - i don't really know what styles etc apply to formio.  but I agree, we have to be super particular with any style override.  between vuetify and bootstrap and formio, there are a lot of implications.  i think we are going to have to accept that not everything will look like the bc gov design system, and any style/class override that must be done (like above) has to be done in the most specific way possible.  the finest grained scope that will correct the issue.

**Review by loneil — APPROVED** [review]

**Pull request merged**